tests/integration_tests/build/test_coverage.py

```diff
@@ -64,35 +64,35 @@ def test_coverage(test_fc_session_root_path, test_session_tmp_path):
     assert len(proc_model) == 1, "Could not get processor model!"
     coverage_target_pct = COVERAGE_DICT[proc_model[0]]
     exclude_pattern = (
-        '${CARGO_HOME:-$HOME/.cargo/},'
-        'build/,'
-        'tests/,'
-        'usr/lib/gcc,'
-        'lib/x86_64-linux-gnu/,'
-        'test_utils.rs,'
+        "${CARGO_HOME:-$HOME/.cargo/},"
+        "build/,"
+        "tests/,"
+        "usr/lib/gcc,"
+        "lib/x86_64-linux-gnu/,"
+        "test_utils.rs,"
         # The following files/directories are auto-generated
-        'bootparam.rs,'
-        'elf.rs,'
-        'mpspec.rs,'
-        'msr_index.rs,'
-        'bindings.rs,'
-        '_gen'
+        "bootparam.rs,"
+        "elf.rs,"
+        "mpspec.rs,"
+        "msr_index.rs,"
+        "bindings.rs,"
+        "_gen"
     )
-    exclude_region = '\'mod tests {\''
+    exclude_region = "'mod tests {'"
```

Comment:
In this case changing `'` into `"` makes sense because it saves us from needing to escape `'`. But is there any particular reason for changing `'` enclosed strings into `"` all over the file? In Python, single-quoted strings and double-quoted strings are the same. 
We should pick one and stick with it (apart from cases when the string contains single or double quotes that need to be escaped).

Reply:
While it doesn't matter in Python, using `"` conforms to the more universal pattern of `"` for strings and `'` for chars.

Also the popular python formatter [`black`](https://github.com/psf/black) uses `"` for strings and this is what I just used out of habit which changed all the strings to use it. I would argue in favor of both, using `"` for strings and `black` for formatting (e.g. `black .`) as I think  is it easier and simpler than `pycodestyle`.

For now I will undo these stylistic changes to be more inline with the current style.

Reply:
`pycodestyle` is what we currently use for python formatting (see [here](https://github.com/firecracker-microvm/firecracker/blob/main/tests/integration_tests/style/test_python.py#L26)). Using `black` would require a full reformat for all python files, and a change in our python-style test, as well. I am not against switching to `black`, but we should first see the differences between the two first and have a discussion about the benefits.

> While it doesn't matter in Python, using " conforms to the more universal pattern of " for strings and ' for chars

I agree. We should settle on one, make the changes in all of the files and maybe document it somewhere as a rule for Firecracker contributing.

Reply:
> Using black would require a full reformat for all python files, and a change in our python-style test, as well.

This would essentially just be running `black .` then skimming and re-testing to check if it broke anything.

Reply:
Yes, I am not saying that it's hard work, I am just saying that complying with one formatter should be done for all files at once and that our python style test should reflect the preferred formatter.